notebooks/intercomparison-covermaps.ipynb

```diff
@@ -0,0 +1,1881 @@
+{
```

Comment:
<p>Nice this looks pretty good!</p><p>I personally prefer flattening control flows in for loops by using <code>continue</code> but you don't necessarily have to take this feedback. Here's how it could look with <code>continue</code> </p><pre class="ql-syntax" spellcheck="false">for dataset in ["cop", "esa", "glad", "harvest_togo", "harvest_kenya", "harvest_tanzania"]:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; if pd.isnull(df[dataset]).all():
            print(f"{dataset} is empty"
            continue
        if len(df[dataset].unique()) &gt; 1:
            print(f"{dataset}: found multiple datasets.")
            continue

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; print(country + ": " + dataset)&nbsp;&nbsp;&nbsp; 
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;clean_df = df[["test_class", dataset]].dropna() # Remove na values
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;if len(clean_df) &lt; 10:
            print(f"{dataset}" has less than 10 examples")
            continue

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;report = classification_report(clean_df["test_class"], temp[dataset], output_dict= True)&nbsp;&nbsp;&nbsp; &nbsp;

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; results[country] = report_to_row(dataset, report, results[country])

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
Oh no this did not come out as nicely outside ReviewNB